.github/workflows/test-spras.yml

```diff
@@ -27,15 +27,16 @@ jobs:
 
   # Runs the test code and Snakemake workflow in the conda environment
   test:
-    name: Run test cases and workflow
+    name: Run tests
     # The Docker images will be pulled in both the docker job and this test job
     runs-on: ${{ matrix.os }}
     strategy:
       matrix:
+        test_group: [1, 2]
```

Comment:
Because this isn't commonly used in workflows that I work with, I would find it helpful to have a comment previewing what these ints will do later.